packages/streaming-image-volume-loader/src/StreamingImageVolume.ts

```diff
@@ -35,8 +35,12 @@ export default class StreamingImageVolume extends BaseStreamingImageVolume {
    */
   public getImageLoadRequests = (priority: number) => {
     const { imageIds } = this;
-    const scalarData = <Types.VolumeScalarData>this.scalarData;
 
-    return this.getImageIdsRequests(imageIds, scalarData, priority);
+    return this.getImageIdsRequests(imageIds, priority);
+  };
+
+  public getImageIdsLoad = () => {
+    const { imageIds } = this;
+    return imageIds;
   };
```

Comment:
getImageIds?

Reply:
It is just below getImageLoadRequests and is not the full list of image ids necessarily, but the ones for being loaded right now, so I'm not quite sure what to call it.  let me know if you have an idea.